### PR TITLE
`<InteractiveModal />:` delete component

### PR DESCRIPTION
**Pull request by cmarin001**
This PR encompasses the removal of the `<InteractiveModal />` component from our library. After careful consideration and evaluation, it's been determined that this component is no longer necessary or has been replaced by other more efficient components. Kindly review the deletion to ensure that no dependencies or references remain in the codebase that might cause unexpected behaviors.

### CODE REVIEW

File: src/components/data/Table/DisplayEntry/index.tsx [hunk 1]
```diff
@@ -1,33 +1,16 @@
-import { InteractiveModal } from "@feedback/InteractiveModal";
 import { useState } from "react";
 import { MdOpenInNew } from "react-icons/md";
-import { IAction } from "../interface";
-import { ITitle } from "../interface";
 
 export interface IEntry {
   id: string;
   [key: string]: any;
```
Comment (Andresbl123) [inline src/components/data/Table/DisplayEntry/index.tsx:6]: Aquí para evitar el `any`, podemos mejor tipar con `string`,  que estaría mas acorde, a la lista de objetos de `entries`

File: src/components/data/Table/stories/Table.stories.tsx [hunk 2]
```diff
@@ -1,8 +1,12 @@
+import { useState, useEffect } from "react";
 import { Table, ITableProps } from "../index";
```
Comment (Andresbl123) [inline src/components/data/Table/stories/Table.stories.tsx:2]: en esta parte sobraría el `index`

Comment (cmarin001) [reply src/components/data/Table/stories/Table.stories.tsx:2]: resuelto

**Review by Andresbl123 — APPROVED** [review]
Cumple requerimentos

**Pull request merged**